{"name": "test_create_task_without_name", "status": "passed", "description": "Проверка создания задачи без имени — должен вернуть ошибку", "steps": [{"name": "Создаем задачу без имени", "status": "passed", "start": 1749925210340, "stop": 1749925210463}], "start": 1749925210339, "stop": 1749925210463, "uuid": "147865b9-0516-4c28-af2e-679b690d8a58", "historyId": "19c5729ffa9951a80e22d57809024d95", "testCaseId": "19c5729ffa9951a80e22d57809024d95", "fullName": "tests.api.test_tasks.TestTasks#test_create_task_without_name", "labels": [{"name": "feature", "value": "Clickup API: Tasks"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "A1"}, {"name": "thread", "value": "37448-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}